assets/src/js/components/fonts.js

```diff
@@ -22,18 +24,14 @@ const WebFont = {
 	 * @return {void}
 	 */
 	loadWebFonts() {
-
 		const WebFontConfig = {
 			google: {
-				families: [
-					'Open Sans:300,400,700'
-				]
-			}
+				families: [ 'Open Sans:300,400,700' ],
```

Comment:
**Suggestion -**

Google recommends self-host Google fonts locally under the theme directory. We can remove this unnecessary package. Instead, you can add a `fonts` directory under assets with the default `Open Sans` font files and Webpack configs.

Reply:
Yep, we can remove this however, the underlying issue may not be the webfont loader, I would like to see how the self-hosted fonts perform in terms of FOUT compare to the loader, also if there's something that we may have missed while using the webfont loader - load the script with prefetch  🤔 

So yep, this change can be done, but keeping this around for further experiments. Or if you have some material on similar tests it would save some time :)

your thoughts?